docs/tutorials/plugin_development.html.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Plugin Developement
+subtitle: How to build a plugin for Danger Kotlin
+layout: guide_kt
+order: 2
+blurb: How do i make a plugin for Danger Kotlin?
+---
+
+## Introduction
+
+Danger provides an sdk for the plugin development. This sdk works with the principle of dependency inversion, that means 
```

Comment:
Why is this sentence trucated, if you want to avoid long paragraphs, should it go on the other line after the comma?